pkg/operator/controller.go

```diff
@@ -271,10 +272,15 @@ func (c *Controller) handler() cache.ResourceEventHandlerFuncs {
 					return
 				}
 			}
-			klog.V(1).Infof("add event to workqueue due to %s (add)", utilObjectInfo(o))
+			obj := o.(metav1.Object)
+			if obj.GetNamespace() == "kube-system" && obj.GetName() != "cluster-config-v1" {
+				return
+			}
+			klog.V(0).Infof("add event to workqueue due to %s (add)", utilObjectInfo(o))
 			c.workqueue.Add(workqueueKey)
 		},
 		UpdateFunc: func(o, n interface{}) {
+			//add event to workqueue due to *v1.ConfigMap, Namespace=kube-system, Name=cluster-config-v1
```

Comment:
What is this comment for?

Reply:
not sure where that came from :(
I will remove it